src/main/scala/com/raphtory/core/components/querymanager/QueryHandler.scala

```diff
@@ -384,6 +392,7 @@ abstract class QueryHandler(
     sentMessageCount = 0
     checkingMessages = false
 
+    QueryTelemetry.totalGraphOperations.inc()
```

Comment:
Similar to above, may be an idea to split out to specify per Job what is running